src/sentry/ownership/grammar.py

```diff
@@ -192,46 +190,46 @@ class Owner(namedtuple("Owner", "type identifier")):
         #team
     """
 
-    def dump(self):
+    def dump(self) -> Mapping[str, str]:
         return {"type": self.type, "identifier": self.identifier}
 
     @classmethod
-    def load(cls, data):
+    def load(cls, data: Mapping[str, Any]) -> "Owner":
         return cls(data["type"], data["identifier"])
 
 
-class OwnershipVisitor(NodeVisitor):
+class OwnershipVisitor(NodeVisitor):  # type: ignore
     visit_comment = visit_empty = lambda *a: None
 
-    def visit_ownership(self, node, children):
+    def visit_ownership(self, node: Any, children: Any) -> Any:
```

Comment:
Are these all just types from parsimonious? It'd be better to use the classes from there if possible